mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -1971,16 +1885,105 @@ async fn connect_loop(
 #[cfg(target_arch = "wasm32")]
 async fn connect_loop(
```

Comment:
In the future, I prefer having the single version of the `connect_loop` that will be transport agnostic. It might be done as part of major planned refactoring (https://github.com/KomodoPlatform/atomicDEX-API/issues/746) that we will hopefully do sooner than later :slightly_smiling_face: